Makefile

```diff
@@ -0,0 +1,12 @@
+PYTHON3=$(shell which python3)
+obj = mygnupg.py
+
+# just run python for test
+runpy:
```

Comment:
Instead of hiding this in the makefile I would recommend adding a section to the README about how to run python in pipenv.

Reply:
Ok, I will change it.
Also, since this pull request merges commit into the wrong branch, I will open another branch to change this.